src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -253,6 +253,7 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
             {
                 Logger.Write(TraceEventType.Verbose, "Shutting down language service");
                 DeletePeekDefinitionScripts();
+                DeleteAgentNotebooks();
                 this.Dispose();
```

Comment:
This should go in the InitializeService method of AgentService, not here

Reply:
Moved